Dining/forms.py

```diff
@@ -53,6 +54,15 @@ def __init__(self, user, date, *args, **kwargs):
             self.fields['association'].initial = available[0].pk
             self.fields['association'].disabled = True
 
+    def clean(self, *args, **kwargs):
+        cleaned_data = super(CreateSlotForm, self).clean(*args, **kwargs)
+        # Check if the time is within time limits
+        serve_time = cleaned_data['serve_time']
+        if serve_time < settings.KITCHEN_USE_START_TIME:
+            self.add_error('serve_time', _("Kitchen can't be used this early"))
```

Comment:
Should be `raise ValidationError()` I think.

Reply:
Nope, this makes the error appear at the specefic field instead of the full form.